format/spec.md

```diff
@@ -631,6 +632,29 @@ When expiring snapshots, retention policies in table and snapshot references are
     2. The snapshot is not one of the first `min-snapshots-to-keep` in the branch (including the branch's referenced snapshot)
 5. Expire any snapshot not in the set of snapshots to retain.
 
+#### Statistics file
+
+Statistics files are valid [Puffin files](../puffin-spec). Statistics are informational. A reader can choose to
+ignore statistics information. Statistics support is not required to read the table correctly.
+
+Statistics files' metadata within `statistics` table snapshot field is a struct with the following fields:
+
+| v2         | Field name                      | Type                                   | Description                                                                                            |
+|------------|---------------------------------|----------------------------------------|--------------------------------------------------------------------------------------------------------|
+| _required_ | **`statistics-path`**           | `string`                               | Path of the statistics file. See [Puffin file format](../puffin-spec).                                 |
+| _required_ | **`file-size-in-bytes`**        | `long`                                 | Size of the statistics file.                                                                           |
+| _required_ | **`file-footer-size-in-bytes`** | `long`                                 | Size of the statistics file's footer. See [Puffin file format](../puffin-spec) for footer definition.  |
```

Comment:
I think this needs to be more clear, since the Puffin footer contains a size. This is not the size from the Puffin footer, it is the size of the entire footer, which is the Puffin footer payload size + 16 (start magic, footer payload size, flags, end magic). Saying this is the "total footer size (not the footer payload size)" should be sufficient.